_includes/js/prettier_changelog.js

```diff
@@ -0,0 +1,73 @@
+var headings = document.querySelectorAll('h2,h3');
+/* Store last level 2 heading */
+var lastHeadingTwo = '';
+/* Regex for ISO-like date strings */
+var dateRE = new RegExp('\\d{2,4}\\-\\d{1,2}\\-\\d{1,2}');
+
+function getHref (element)
+{
+  /*
+   * get the href value without the # and return as a string.
+   */
+  return element.firstChild.attributes.href.value.match(/[^\#]+/).toString();
+}
+
+function setHref (element, newValue)
+{
+  /*
+   * set a new href value.
+   */
+  element.firstChild.attributes.href.value = newValue;
+  return element;
+}
+
+function fixAnchor (element, newValue)
+{
+  /*
+   * replace the anchor that sits just before the heading in the DOM.
+   */
+  element.previousSibling.id = newValue;
+  return element;
+}
+
+function prettyDate (heading, dateRegex)
+{
+  /* Change ISO-like date to human-readable date. */
+  var date_options = {
```

Comment:
We are hard-coding the format. if the first param for `toLocaleDateString`is `undefined` it will check the browser, which is why I hard-coded it for now.

Reply:
we're using JS rather than hard-coding so we can just drop the changelog entries in, rather than manually change them each time. I hope to create a plugin for the next system which will change and hard-code the long-date at build-time.